infra/Makefile

```diff
@@ -0,0 +1,10 @@
+default: plan
```

Comment:
why not just reuse this https://github.com/HumanCellAtlas/fusillade/blob/b5ec9ed8fa3a45fc0f449ed01b828c2bbafc80a5/infra/Makefile#L31-L48

Reply:
What is missing in my version of the makefile?